public/index.html

```diff
@@ -13,7 +13,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
```

Comment:
If you'd like, an OG:tag could be added to increase the share'ability of the portfolio! 💯

Reply:
Thanks! Great tips! :)